### PR TITLE
Support/live 8503 remove ftx wyre code

### PR DESCRIPTION
**Pull request by ggilchrist-ledger**
### 📝 Description

Removing all code related to FTX, Wyre and the accompanying KYC, MFA and auth code that came with them.

Tested full swaps on:
- [x] LLD
- [x] LLM iOS
- [x] LLM Android

### ❓ Context

- **Impacted projects**: `` 
- **Linked resource(s)**: https://ledgerhq.atlassian.net/browse/LIVE-8503

### ✅ Checklist

- [x] **Test coverage** 
- [ ] **Atomic delivery** 
- [x] **No breaking changes** 

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

File: apps/ledger-live-mobile/src/screens/Swap/Form/Connect.tsx [hunk 1]
```diff
@@ -22,7 +22,7 @@ export function Connect({
   provider?: string;
 }) {
   const [device, setDevice] = useState<Device>();
-  const [result] = useState();
+  const [result] = useState(); // WTF
```
Comment (ggilchrist-ledger) [inline apps/ledger-live-mobile/src/screens/Swap/Form/Connect.tsx:25]: Can you help me figure out what's going on here @sarneijim ? I've not seen a useState get used like this before

Comment (sarneijim) [reply apps/ledger-live-mobile/src/screens/Swap/Form/Connect.tsx:25]: Second param is optional, as `setResult` is not going to be used, you do not need to set up. (Similar what we saw this morning second argument is not required).
```
const [result, functionToSetNotUsed] = useState(); 
===
const [result] = useState();

**Review by sarneijim — COMMENTED** [review]
https://tenor.com/bgUEh.gif

File: .changeset/cuddly-phones-hide.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"ledger-live-desktop": patch
+---
+
+Removed FTX/Wyre auth related code for LLD
```
[the remark this answers is not in the document]
Comment (ggilchrist-ledger) [reply .changeset/cuddly-phones-hide.md:5]: Should I delete this one?

File: apps/ledger-live-mobile/src/screens/Swap/Form/Connect.tsx (hunk 1 above)
Comment (ggilchrist-ledger) [reply apps/ledger-live-mobile/src/screens/Swap/Form/Connect.tsx:25]: `setResult` is passed in as a function to the component and used when hiding the modal. Maybe it's fine, just looks a bit funny

File: .changeset/cuddly-phones-hide.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (sarneijim) [reply .changeset/cuddly-phones-hide.md:5]: Multiple paths, choose your destiny :)
You can reuse it, modify the file manually. 
Create a new one with the other "libs".
Remove this file and add a new one with all the libs.

**Review by gre — APPROVED** [review]

**Review by sarneijim — APPROVED** [review]

**Pull request merged**